ui-speedspacechart/src/components/layers/AxisLayerY.tsx

```diff
@@ -4,18 +4,18 @@ import type { Store } from '../../types/chartTypes';
 import { drawGridY } from '../helpers/drawElements/gridY';
 import { useCanvas } from '../hooks';
 
-type AxisLayerYProps = {
+type GridYProps = {
```

Comment:
We try to keep the props type name the same as the component. We should either rename the component `GridY` to match the type and the id or keep these as they were before.